lib/tizen_cache.dart

```diff
@@ -45,20 +47,30 @@ class TizenDevelopmentArtifact implements DevelopmentArtifact {
   static const DevelopmentArtifact tizen = TizenDevelopmentArtifact('tizen');
 }
 
-class TizenCache extends FlutterCache {
+class TizenCache extends Cache {
   TizenCache({
     required Logger logger,
     required FileSystem fileSystem,
     required Platform platform,
     required OperatingSystemUtils osUtils,
     required ProcessManager processManager,
-    required super.projectFactory,
+    required FlutterProjectFactory projectFactory,
   }) : super(
           logger: logger,
-          fileSystem: fileSystem,
           platform: platform,
+          fileSystem: fileSystem,
           osUtils: osUtils,
+          artifacts: <ArtifactSet>[],
         ) {
+    registerArtifact(FontSubsetArtifacts(this, platform: platform));
```

Comment:
This line could be removed as mentioned in the above.

Reply:
done!